src/bp/ui-studio/src/web/views/FlowBuilder/style.scss

```diff
@@ -1,11 +1,10 @@
 .diagram {
   height: calc(100%);
-  background-color: white;
+  background-color: #f1f1f1;
 }
 
 .diagram :global(.srd-diagram) {
   height: calc(100%);
-  background: url('assets/ui-studio/public/img/grid.png') repeat;
```

Comment:
The diagram should still have the grid pattern

Reply:
really ? when I look at the empty state in provided designs I can't see any grid

Reply:
this isn't the empty state, this is the diagram, there's no grid anywhere now

Reply:
I don't think these changes should be done until we work on the workflow, it's weird AF with the current nodes